actgen/plot.py

```diff
@@ -80,22 +80,28 @@ def preprocess_training_rewards(directory):
 	return time_step, avg_rewards, ci
 
 
-def plot_training_data(normal, oracle, no_duplicate, exp_name, data_type):
+def plot_training_data(normal, oracle, no_duplicate, rand, rand_oracle, exp_name, data_type):
```

Comment:
This code is getting a little difficult to read. Is there an easy way to clean it up?

Reply:
yea I agree it's getting kind of confusing. I will refactor all of `plot.py` to use pandas and seaborn soon